frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -44,6 +44,27 @@ const DropdownMenuProps = {
   getOptionSelected,
 };
 
+const ANALYTIC_MAPPING: {
```

Comment:
moved this constant out to the top, so we don't re-generate the object at each render